client/post-editor/editor-sidebar/index.jsx

```diff
@@ -77,6 +77,7 @@ const EditorSidebar = ( {
 							selectedRevisionId={ selectedRevisionId }
 							selectRevision={ selectRevision }
 							siteId={ site.ID }
+							type={ type }
```

Comment:
Passing this as a prop is arguably a bit redundant, because `siteId` and `postId` props should be sufficient for the component to be able to look up this information from state on its own via `connect`. [EditorDocumentHead](https://github.com/Automattic/wp-calypso/blob/b3fc15c/client/post-editor/editor-document-head/index.jsx#L65) is similar here (though more extreme in that it doesn't accept any props) where it determines the `type` using the `getEditedPostValue` selector.